apps/desktop/desktop/src/main/java/bisq/desktop/main/content/authorized_role/mediator/MediationCaseHeader.java

```diff
@@ -131,7 +131,7 @@ void onToggleOpenClose() {
         void onLeaveChannel() {
             if (dontShowAgainService.showAgain(MEDIATOR_LEAVECHANNEL_WARNING)) {
                 new Popup().warning(Res.get("authorizedRole.mediator.leaveChannel.warning"))
-                        .dontShowAgainId(MEDIATOR_LEAVECHANNEL_WARNING.getKey())
+                        .dontShowAgainId(MEDIATOR_LEAVECHANNEL_WARNING)
```

Comment:
CamelCase to all upper case would result in MEDIATOR_LEAVE_CHANNEL_WARNING.
Same for other such cases.